### PR TITLE
Added QA authorities as search options.

### LINKED ISSUE #1377: Add search choices to search tab
* Update search tab to match design described in https://github.com/LD4P/sinopia/issues/255
* Provide configuration of QA authorities to be listed. This can use the same / similar configuration approach as `lookupConfig.json`.
* Note that one of the search options should be "Sinopia" and should perform a search of Sinopia as it currently does.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #1377

### CODE REVIEW

**Comment by justinlittman** [comment]
Screen Shot 2019-09-30 at 10 47 52 PM
Screen Shot 2019-09-30 at 10 48 05 PM

File: static/searchConfig.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
+[
+  {
+    "label": "SHAREVDE ALBERTA",
```
Comment (jcoyne) [inline static/searchConfig.json]: https://media1.giphy.com/media/ii8rlXOlRFp5LKAVUR/giphy.gif

Comment (justinlittman) [reply static/searchConfig.json]: Or that he copied the existing config ...

File: src/components/search/Search.jsx [hunk 2]
```diff
@@ -43,27 +58,41 @@ const Search = (props) => {
...
-            </FormGroup>
-          </Form>
-        </Row>
-        <SearchResults {...props} key="search-results" />
-        <SearchResultsPaging {...props} pageSize="1"/>
-        <SearchResultsMessage />
-      </Grid>
+      <div className="row">
+        <form className="form-inline" onSubmit={handleSubmit}>
+          <div className="form-group col-md-4" style={{ paddingRight: '5px' }}>
```
Comment (jcoyne) [inline src/components/search/Search.jsx]: form-group elements should not have grid styles (e.g. `col-md-4`)
```

$ bootlint << EOF
>
> <form class="form-inline">
>   <div class="form-group col-md-3"></div>
> </form>
> EOF
...
<stdin>:3:3 E014 Columns (`.col-*-*`) can only be children of `.row`s or `.form-group`s
```

File: src/components/search/Search.jsx [hunk 3]
```diff
@@ -43,27 +58,41 @@ const Search = (props) => {
...
+            <label htmlFor="searchType" style={{ marginRight: '5px' }}>Search</label>
+            <select className="form-control" id="searchType"
+                    value={authority} style={{ paddingRight: '0px', width: '85%' }}
```
Comment (jcoyne) [inline src/components/search/Search.jsx]: In an `.form-inline` aren't select's using `width: auto`?  Why are you diverging from that?

File: src/components/search/Search.jsx [hunk 4]
```diff
@@ -43,27 +58,41 @@ const Search = (props) => {
...
+                    onChange={ event => setAuthority(event.target.value) }
+                    onBlur={ event => setAuthority(event.target.value) }>
+              <option value="sinopia">Sinopia</option>
+              {options}
+            </select>
+          </div>
+          <div className="form-group col-md-6" style={{ paddingLeft: '5px' }}>
+            <div className="input-group" style={{ width: '100%' }}>
```
Comment (jcoyne) [inline src/components/search/Search.jsx]: Isn't the width always 100%?

File: src/components/search/Search.jsx [hunk 5]
```diff
@@ -43,27 +58,41 @@ const Search = (props) => {
...
+            <label htmlFor="searchType" style={{ marginRight: '5px' }}>Search</label>
+            <select className="form-control" id="searchType"
+                    value={authority} style={{ paddingRight: '0px', width: '85%' }}
+                    onChange={ event => setAuthority(event.target.value) }
+                    onBlur={ event => setAuthority(event.target.value) }>
+              <option value="sinopia">Sinopia</option>
+              {options}
+            </select>
+          </div>
+          <div className="form-group col-md-6" style={{ paddingLeft: '5px' }}>
```
Comment (jcoyne) [inline src/components/search/Search.jsx]: can't use columns here.

File: src/components/search/Search.jsx [hunk 6]
```diff
@@ -43,27 +58,41 @@ const Search = (props) => {
...
+            <div className="input-group" style={{ width: '100%' }}>
+              <label className="sr-only" htmlFor="searchInput">Query</label>
+              <input id="searchInput" type="text" style={{ width: '100%' }} className="form-control"
+                     onChange={ event => setQueryString(event.target.value) }
+                     onKeyPress={handleKeyPress} />
+              <span className="input-group-btn">
+                <button className="btn btn-default" type="submit"><FontAwesomeIcon className="fa-search" icon={faSearch} /></button>
```
Comment (jcoyne) [inline src/components/search/Search.jsx]: Need aria (or `.sr-only`) text for screen readers

File: src/components/search/Search.jsx [hunk 7]
```diff
@@ -43,27 +58,41 @@ const Search = (props) => {
...
+                    onChange={ event => setAuthority(event.target.value) }
+                    onBlur={ event => setAuthority(event.target.value) }>
+              <option value="sinopia">Sinopia</option>
+              {options}
+            </select>
+          </div>
+          <div className="form-group col-md-6" style={{ paddingLeft: '5px' }}>
+            <div className="input-group" style={{ width: '100%' }}>
+              <label className="sr-only" htmlFor="searchInput">Query</label>
+              <input id="searchInput" type="text" style={{ width: '100%' }} className="form-control"
```
Comment (jcoyne) [inline src/components/search/Search.jsx]: I believe `.form-control` already sets `width: 100%`

File: static/searchConfig.json (hunk 1 above)
Comment (jcoyne) [reply static/searchConfig.json]: But it's hard to read. Can we improve it by using normal case?

File: src/components/search/Search.jsx (hunk 7 above)
Comment (jcoyne) [inline src/components/search/Search.jsx]: `type` should be `url` See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url

File: static/searchConfig.json (hunk 1 above)
Comment (justinlittman) [reply static/searchConfig.json]: @michelleif might want these to say something entirely different, e.g., different case, different names, no SHAREVDE. Would prefer to handle in a separate PR in which she can edit as she feels appropriate.

**Comment by jcoyne** [comment]
@justinlittman need to re-eslint this.

**Pull request merged**